fenix/buildSrc/src/main/java/FenixDependencies.kt

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // If you ever need to force a toolchain rebuild (taskcluster) then edit the following comment.
-// FORCE REBUILD 2023-05-04
+// FORCE REBUILD 2023-05-05
```

Comment:
FWIW, this isn't needed if you're making actual changes to the dependency list anyway. Same for the AC one.

Reply:
gotcha, I'll remember for next time